lib/auth.js

```diff
@@ -1067,6 +1066,164 @@ function authorize(req, res, next) {
     next();
 }
 
+/*
```

Comment:
this is *completely* copied & pasted from libmanta

Reply:
Would it make sense to (a) pull all the libmanta pieces that are used into a separate `./libmantalite.js` to highlight the connection and/or (b) have a comment pointing out the bits that are pulled directly from libmanta?

Reply:
I like the idea of a `libmantalite.js`... i'll make that change.